cogs/tickets.py

```diff
@@ -190,6 +190,12 @@ async def close(self, ctx, *, reason: str = None):
         await log_channel.send(embed=embed,
                                file=discord.File(f"transcripts/{ctx.channel.name}.html", "transcript.html"))
 
+    @commands.command(hidden=True, aliases=['sc', 'staffclaim'])
+    @staff_or_trainee
+    async def staff_claim(self, ctx, arg):
```

Comment:
It would be better to allow a channel as an optional argument before the name, using `ctx.channel` as the default if none is provided.